### PR TITLE
fix: text not recognize type

### PR DESCRIPTION
**Pull request by huydosgtech**
## Check List
- [ ] Your code builds clean without any errors or warnings.
- [ ] Check coding convention.
- [ ] Updated Storybook components and stories to reflect any changes made to the UI.
- [ ] Test covers all edge cases and coverage is greater than or equal to 80%.

## Proof of Completeness
Screenshot changes introduced" width="250" />
OR
Video link

### CODE REVIEW

File: src/components/Text/Text.tsx [hunk 1]
```diff
@@ -1,23 +1,74 @@
...
   fontWeight?: TextStyle['fontWeight']
+  /**
+   * Custom font family
+   */
   fontFamily?: string
-} & TextProperties
+  /**
+   * Custom text style
+   */
+  textStyle?: StyleProp<TextStyle>
```
Comment (honghoangsts) [inline src/components/Text/Text.tsx:26]: can we use style props from default

**Review by honghoangsts — CHANGES_REQUESTED** [review]
